client/src/amplify-ui/code-block-switcher/code-block-switcher.style.ts

```diff
@@ -0,0 +1,41 @@
+import {css} from "emotion";
+
+export const hostStyle = css`
+  display: block;
+  margin-bottom: 1.5rem;
+
+  > pre {
+    display: none;
+  }
+
+  amplify-code-block {
+    border-top-right-radius: 0;
+    border-top-left-radius: 0;
+    margin-top: 0;
+  }
+`;
+
+export const activeTabStyle = css`
+  font-weight: 700;
+`;
+
+export const tabStyle = css`
+  color: var(--color-white);
```

Comment:
Remember to use variables here and below

Reply:
Fixed